express_cut/express_service/permissions.py

```diff
@@ -131,3 +131,20 @@ def duration_permissions(self, request, reservation):
             return True
         return False
 
+
+class FeedbackPermissions(Permissions):
+
+    def POST_permissions(self, request, obj):
+        if request.user.is_authenticated and self.has_client_permission(request) and request.user.pk == obj.customer.pk:
+            return True
+        return False
+
+    def GET_permissions(self, request, obj):
+        if request.user.is_authenticated and self.has_client_permission(request) and request.user.pk == obj.customer.pk:
+            return True
+        elif request.user.is_authenticated and self.has_manager_permission(request):
+            return True
+        elif request.user.is_authenticated and self.has_stylist_permission(request):
```

Comment:
Think stylist should have permission is it is part of the reservation. Dont know if other stylists should be able to view other feedback

Reply:
Ok, will change.